packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode } from 'react';
+import { use100vh } from 'react-div-100vh';
+
+type TDiv100vhContainerProps = {
+    id?: string;
+    children: ReactNode;
+    height_offset?: string;
+    is_bypassed: boolean;
+    is_disabled: boolean;
+    max_height_offset?: string;
+    className?: string;
+    max_autoheight_offset?: string;
+};
+
+const Div100vhContainer = ({
+    children,
+    className,
+    is_bypassed = false, // to bypass usage of component altogether, pass it
+    is_disabled = false, // To manually remove use100vh() calculation and revert to default browser calculation pass it
+    id,
+    height_offset,
+    max_autoheight_offset,
+}: TDiv100vhContainerProps) => {
```

Comment:
I think we can remove children from types and add here following: PropsWithChildren<TDiv100vhContainerProps>

you can have a look on this article: https://www.newline.co/@bespoyasov/how-to-define-props-with-children-in-react-typescript-app--56bd18be

Reply:
yes I saw that recently in your PR. will change it :+1:

Reply:
@yauheni-kryzhyk-deriv but in my component Children is not optional. when we use `PropsWithChildren`, TS consider the children as optional.